### DISCUSSION TITLE
Not show Blue checks not working

### DISCUSSION THREAD

**Discussion by ghurty** [Schwartzblat/WhatsApp-Web-Plus#39] [General]
Hi,
This extension is great, but the blue checks show right away as soon as I read the message.

Thank you

**Comment by Schwartzblat** [comment]
You need to cancel it from the WhatsApp privacy settings.

**Reply by ghurty** [reply]
So in whatsapp, I have to say not to show the bluechecks? But then would I be able to see someone else's bluechecks?

**Reply by Schwartzblat** [reply]
Yes, you will receive blue ticks as usual in your web client.
